src/components/Editor/ClientCursor.ts

```diff
@@ -1,88 +1,103 @@
 import invert from 'invert-color';
 
-const markerRemoveMap = new Map<string, number>();
-
-const duration = 0.2;
-const delay = 2;
-const SECOND = 1000;
-const REMOVE_TIME = (duration + delay) * SECOND;
+type actorId = string; // used in yorkie-sdk-js client
+type nameRemoveTimerID = ReturnType<typeof setTimeout>;
```

Comment:
Why did you give the type a name?

Reply:
I put it in to give meaning. Is there a good way?? 🧐

Reply:
I think you can give meaning to variable names that use this type.

Reply:
I didn't understand exactly. Could you explain it in detail?? 🙇‍♀️

Reply:
In the example usage below, it looks like the variable name and type name are duplicated.
```
nameRemoveTimeMap: Map<ActorId, NameRemoveTimerID>;
```

So my suggestion was to change it like this. 😉

```
class ClientCursor {
  timerMap: Map<string, ReturnType<typeof setTimeout>>;
  constructor(actorId: string, color: string) { ... }
```
or
```
class ClientCursor {
  actorId: string;
  timerIdMap: Map<string, ReturnType<typeof setTimeout>>;
...
```